fah_alchemy/base/api.py

```diff
@@ -79,7 +83,9 @@ def validate_scopes_query(
         ):
 
             # match (query_org == token_org) then (query_campaign == token_campaign) then (query_proj == token_proj)
-            if not (query_field == target_field or query_field is None):
+            if not (
+                query_field == target_field or query_field is None or query_field == "*"
```

Comment:
We don't need to test for `'*'` here, as the field will actually be `None` if it's the wildcard value. We use `'*'` in the string representation, but in a `Scope` object this is actually represented as `None`.

This ends up having nice consequences when we use `Scope` objects to query Neo4j, since for a property dict keys with values of `None` end up not being included as query params, requiring us to do little to no processing in the query code.

Reply:
I've changed `Scope._validate_component` to turn '*' into `None` when given as a component on `init`.

Reply:
Okay makes sense to me to not to have "*" and instead cast to `None`